packages/experiments-realm/crm/contact.gts

```diff
@@ -0,0 +1,392 @@
+import StringField from 'https://cardstack.com/base/string';
+import NumberField from 'https://cardstack.com/base/number';
+import {
+  Component,
+  CardDef,
+  FieldDef,
+  field,
+  contains,
+} from 'https://cardstack.com/base/card-api';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+import { fn } from '@ember/helper';
+import { IconButton, RadioInput, Pill } from '@cardstack/boxel-ui/components';
+import Mail from '@cardstack/boxel-icons/mail';
+import Phone from '@cardstack/boxel-icons/phone';
+import BrandTwitter from '@cardstack/boxel-icons/brand-twitter';
+import BrandLinkedin from '@cardstack/boxel-icons/brand-linkedin';
+import Building from '@cardstack/boxel-icons/building';
+import HeartHandshake from '@cardstack/boxel-icons/heart-handshake';
+import TargetArrow from '@cardstack/boxel-icons/target-arrow';
+
+export interface LooseyGooseyData {
+  index: number;
+  label: string;
+  color?: string;
+}
+
+export class LooseGooseyField extends FieldDef {
+  @field index = contains(NumberField); //sorting order
+  @field label = contains(StringField);
+  static values: LooseyGooseyData[] = []; //help with the types
```

Comment:
u can extend LooseyGooseyData to include what you need with icons and update it on the card itself. I prefer if everything is in one place